docs/developer/payments.mdx

```diff
@@ -952,6 +952,52 @@ and its amount is added to `transaction.chargedAmount`. Finally, the amount of t
 
 ---
 
+### Stored payment methods
+
+A stored payment method is a payment method that can be used later by customers without providing all payment details.
+Saleor uses synchronous webhooks to notify the payment app about actions (such as list-stored-payment-method) requested by the customer.
```

Comment:
I would add a link to `list-stored-payment-method`: https://saleor-docs-1juwt4dfh-saleorcommerce.vercel.app/docs/3.x/developer/extending/webhooks/synchronous-events/stored-payment-method#list-stored-payment-methods